File: Java Code/Basics/Mensuration/Cone.java
import java.util.*;

public class Cone{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    float PI=22.0f/7.0f;
    System.out.println("For the Cone :-");
    System.out.print("Enter radius : ");
    float r=sc.nextFloat();
    System.out.print("Enter height : ");
    float h=sc.nextFloat();
    float l=(float)Math.sqrt(r*r+h*h);             //  l = Slant height
    float LSA=PI*r*l;
    float TSA=PI*r*(l+r);
    float V=PI*r*r*h/3;
    System.out.println("Lateral Surface Area : "+LSA);
    System.out.println("Total Surface Area : "+TSA);
    System.out.println("Volume : "+V);
  }
}

File: Javascript Code/Mensuration/Cone.js
const PI=3.14;
let r=parseInt(prompt("Enter radius of the cone : "));
let h=parseInt(prompt("Enter height of the cone : "));
let l=Math.sqrt(r*r+h*h);
console.log(`Volume : ${(PI*r*r*h/3).toFixed(2)} cubic units.`);
console.log(`Lateral Suface Area : ${(PI*r*l).toFixed(2)} sq units.`);
console.log(`Total Surface Area : ${(PI*r*(l+r)).toFixed(2)} sq units.`);